src/fs/FileSystemRegistry.ts

```
@@ -135,6 +151,108 @@ export class FileSystemRegistry extends SynchronousRegistry implements IRegistry
+
+    const signersPath = path.join(this.uri, this.getSignersPath());
+    if (!fs.existsSync(signersPath)) return null;
+
+    const signerFiles = this.listFiles(signersPath).filter(
+      (f) => f.endsWith('.yaml') || f.endsWith('.yml'),
+    );
+
+    if (signerFiles.length === 0) return null;
+
+    // Merge all signer config files
+    let mergedConfig: SignerConfiguration = {};
```

ESLint

[error] 'mergedConfig' is never reassigned. Use 'const' instead.

(prefer-const)